utils/data.py

```
@@ -1,8 +1,10 @@
 import json
 import os
-import logging
+import logging   #로그 남기기
 import shutil
-import tempfile
+import tempfile   #임시파일 만들기(저장 안정성을 위해)
+from typing import Optional, Any, Dict, List
```

Ruff (0.14.13)

`typing.Dict` is deprecated, use `dict` instead

(UP035)